src/main/scala/com/databricks/spark/sql/perf/tpcds/GenTPCDSData.scala

```diff
@@ -0,0 +1,73 @@
+package com.databricks.spark.sql.perf.tpcds
+
+import org.apache.spark.sql.SparkSession
+
+case class GenTPCDSDataConfig(
+    master: String = "local[*]",
+    dsdgenDir: String = null,
+    scaleFactor: String = null,
```

Comment:
Can the scale factor `null`?

Reply:
No, we can use GenTPCDSDataConfig [here](https://github.com/databricks/spark-sql-perf/pull/196/files#diff-4bd69f94d0d482b305f029dc8933b287862a2b0891f87cda08488b1b1862d940R58) with null values, otherwise:
```
[error] /Users/yumwang/opensource/spark-sql-perf/src/main/scala/com/databricks/spark/sql/perf/tpcds/GenTPCDSData.scala:58: not enough arguments for method apply: (master: String, dsdgenDir: String, scaleFactor: String, location: String, format: String)com.databricks.spark.sql.perf.tpcds.GenTPCDSDataConfig in object GenTPCDSDataConfig.
[error] Unspecified value parameter dsdgenDir.
[error]     parser.parse(args, GenTPCDSDataConfig()) match {
[error]                                          ^
[error] /Users/yumwang/opensource/spark-sql-perf/src/main/scala/com/databricks/spark/sql/perf/tpcds/GenTPCDSData.scala:60: type mismatch;
[error]  found   : Any
[error]  required: com.databricks.spark.sql.perf.tpcds.GenTPCDSDataConfig
[error]         run(config)
[error]             ^
[error] two errors found

```

Reply:
Another case:
https://github.com/databricks/spark-sql-perf/blob/e1e1365a8795b76a796dab3a51259be9343c80de/src/main/scala/com/databricks/spark/sql/perf/RunBenchmark.scala#L28